modules/login/php/requestaccount.class.inc

```diff
@@ -269,7 +269,67 @@ class RequestAccount extends \NDB_Form
             }
         }
 
+        // Verify reCAPTCHA
+        $err = [];
+        if (isset($values['g-recaptcha-response']) && isset($reCAPTCHAPrivate)) {
```

Comment:
@johnsaigle why are the recptcha changes here ? bad rebase?

Reply:
Yep. I'll fix it